caluma/caluma_workflow/events.py

```diff
@@ -0,0 +1,12 @@
+from django.dispatch import Signal
+
+new_work_item = Signal(providing_args=["work_item", "info"])
+save_work_item = Signal(providing_args=["work_item", "info"])
+create_work_item = Signal(providing_args=["work_item", "info"])
+cancel_work_item = Signal(providing_args=["work_item", "info"])
+complete_work_item = Signal(providing_args=["work_item", "info"])
+skip_work_item = Signal(providing_args=["work_item", "info"])
+complete_case = Signal(providing_args=["case", "info"])
+save_case = Signal(providing_args=["case", "info"])
+start_case = Signal(providing_args=["case", "info"])
```

Comment:
Another thing: I'd remove the `info` bit from the event. We explicitly want to de-couple the events from the request, so this should be omitted in my opinion.

Reply:
Implemented as discussed on the phone.